### ISSUE TITLE
2013 Mac Pro 4k @ 60hz

### ISSUE THREAD

**Issue by pistolsx1979** [Floris497/mac-pixel-clock-patch-V2#97]
Hi Floris, I wonder if you can help me with an issue I have had since upgrading to OSX Sierra. I have a Samsung UHD TV which I have had running at 60hz using your patch and switchresX but since the upgrade to Sierra my screen says it will no longer support 60hz.  I have applied the coredisplay patcher, it says Sip is disabled and (otool) detected patched core display on OS X 10.12.1 but when i try to select the screen resolution in system prefs the screen tells me the resolution is unsupported. Any ideas where I am going wrong here?

**Comment by Floris497** [comment]
HDMI? Using any adapter?

**Comment by pistolsx1979** [comment]
Hi Floris,

I am running from displayport on the mac to an HDMI 2.0 adapter which has worked fine until now. The HDMI cable is 2.0 also. I managed to get SwitchresX to show the resolution for 60hz but when i try to select it the screen tells me it doesnt support the res although i know it does! very frustrating!

**Comment by Floris497** [comment]
It's weird that the screen tells you it doesn't work.. right hdmi port, some 📺 only support it on a single port.

**Comment by pistolsx1979** [comment]
yes it is really strange, i haven't changed the port that I use on the TV, I also tried different ports on my mac but same issue.

**Comment by Floris497** [comment]
Then i have no idea.. sorry if its on tge tv side..

**Comment by pistolsx1979** [comment]
Thanks for getting back to me, worst thing is I have had it working fine for months, and this Sierra update has caused the issue :(

**Comment by Floris497** [comment]
Could you try and see if it works again in 10.11? Also can you check if patch worked good?

**Comment by Floris497** [comment]
Are you on 10.12.1?

**Comment by pistolsx1979** [comment]
yes i am on the latest 10.12.1 OS X , i dont think i can get back down to 10.11 very easily

**Comment by pistolsx1979** [comment]
Hey Floris, I think i have sorted it now, I actually ran this code in terminal sudo defaults write /Library/Preferences/com.apple.windowserver.plist DisplayResolutionEnabled -bool true
and now it appears i have 60hz again!

**Comment by Floris497** [comment]
That one is only to enable HiDPI I think?

**Comment by CKDigital112** [comment]
I am having the same issue with my Mac Pro (Late 2013) with LG 4K Monitor, HDMI cable through adapter into Thunderbolt 2. Tried the CoreDisplay patch and it did not work. I am on 10.12.2

**Comment by Floris497** [comment]
Is your adapter capable of HDMI2.0? Else you can't get above 4k60Hz

**Comment by CKDigital112** [comment]
[https://www.bhphotovideo.com/c/product/1133809-REG/startech_mdp2hd4ks_mini_displayport_to_hdmi_adapter.html](url)

This is the adaptor. What's weird is that it actually started showing 4K resolution after I wrote here yesterday. Now this morning it's not working in 4K.

**Comment by Floris497** [comment]
Oh I meant 4k@30Hz in mu last response, your cable seems to only be able to do 30Hz
